lib/SYCL/inSPIRation.cpp

```diff
@@ -238,8 +271,24 @@ struct inSPIRation : public ModulePass {
     for (auto &F : M.functions()) {
       // Only consider definition of SYCL kernels
       // \todo Put SPIR calling convention on declarations too
-      if (!F.isDeclaration() && sycl::isKernel(F))
-        kernelSPIRify(F);
+      if (!F.isDeclaration()) {
```

Comment:
Useless `{`

Reply:
Otherwise, not clear why you changed the syntax...

Reply:
Since it causes a warning, I put the { for line 274. There is a chance to have ambiguous syntax for if.